modular-docs-manual/files/TEMPLATE_CONCEPT_concept-explanation.adoc

```diff
@@ -2,29 +2,43 @@
 //
 // <List assemblies here, each on a new line>
 
-// Base the file name and the ID on the module title. For example:
-// * file name: my-concept-module-a.adoc
-// * ID: [id="my-concept-module-a_{context}"]
-// * Title: = My concept module A
+////
+Base the file name and the ID on the module title. For example:
+* file name: con-my-concept-module-a.adoc
+* ID: [id="con-my-concept-module-a_{context}"]
+* Title: = My concept module A
+////
 
-// The ID is used as an anchor for linking to the module. Avoid changing it after the module has been published to ensure existing links are not broken.
-[id="concept-explanation_{context}"]
-// The `context` attribute enables module reuse. Every module's ID includes {context}, which ensures that the module has a unique ID even if it is reused multiple times in a guide.
-= Concept explanation
-//In the title of concept modules, include nouns or noun phrases that are used in the body text. This helps readers and search engines find the information quickly.
-//Do not start the title of concept modules with a verb. See also _Wording of headings_ in _The IBM Style Guide_.
+////
+The ID is used as an anchor for linking to the module. Avoid changing it after the module has been published to ensure existing links are not broken.
+ ////
 
-A short introductory paragraph is required for the concept module.
-It will provide an overview of the module.
+[id="con-my-concept-module-a_{context}"]
+
+////
+The `context` attribute enables module reuse. Every module's ID includes {context}, which ensures that the module has a unique ID even if it is reused multiple times in a guide.
+////
+
+= My concept module A
+////
+In the title of concept modules, include nouns or noun phrases that are used in the body text. This helps readers and search engines find the information quickly. Do not start the title of concept modules with a verb. See also _Wording of headings_ in _The IBM Style Guide_.
+////
+
+[role="_abstract"]
+A short introductory paragraph that provides an overview of the module is required.
```

Comment:
```suggestion
A short introductory paragraph that provides an overview of the module.
```

Reply:
I rewrote:
Write a short introductory paragraph that provides an overview of the module.